resources/client-extension-resources.json

```diff
@@ -1,4 +1,37 @@
 [
+  {
+    "description": "Notification Type Springboot Partial",
+    "name": "notification-type-springboot",
+    "type": "partial",
+    "template": "service-springboot",
+    "args": [
+      {
+        "name": "id",
+        "description": "Id of notification type client extension",
+        "default": "my-notification-type"
+      },
+      {
+        "name": "notificationTypeName",
+        "description": "Prefix of the NotificationType resource: ${notificationTypeName}NotificationType.java",
+        "default": "My"
+      },
+      {
+        "name": "package",
+        "description": "This is a java package 'com.company.service'",
+        "default": "com.company.service"
+      },
+      {
+        "name": "packagePath",
+        "description": "This is a java package path 'com/liferay/service'",
+        "default": "com/company/service"
+      },
+      {
+        "name": "resourcePath",
+        "description": "HTTP POST resource path for the notificationType to invoke",
+        "default": "/mynotificationtype/send"
```

Comment:
I think this might be more informative as `/receive` instead of `/send` since this code is in fact just receiving the message?

Reply:
I was thought about an endpoint that will send a notification.

endpoint: `/whatsapp/send`,
result: It should send a notification through the Whatsapp

Reply:
fair enough!